tests/unit/src/main/scala/bill/Bill.scala

```diff
@@ -374,10 +376,18 @@ object Bill {
     }
     override def buildTargetScalaTestClasses(
         params: ScalaTestClassesParams
-    ): CompletableFuture[ScalaTestClassesResult] = ???
+    ): CompletableFuture[ScalaTestClassesResult] =
+      Future.successful(new ScalaTestClassesResult(List.empty.asJava)).asJava
     override def buildTargetScalaMainClasses(
         params: ScalaMainClassesParams
-    ): CompletableFuture[ScalaMainClassesResult] = ???
+    ): CompletableFuture[ScalaMainClassesResult] = {
+      params.getTargets().asScala.toList match {
+        case List(break, time) if break.getUri == "break" =>
+          sleepBeforePingResponse = Try(Duration(time.getUri)).toOption
+        case _ =>
+      }
```

Comment:
This is super hacky but I'm not sure how to do it better.

Reply:
That's fine, it's only used for testing